rpc/transport/ws/server.go

```diff
@@ -31,21 +33,23 @@ type serverWebSocketTransport struct {
 func NewWebSocketTransportAsServer(port string) (*serverWebSocketTransport, error) {
 	wsc := &serverWebSocketTransport{port: port, notificationListeners: safesync.Map[chan []byte]{}}
 
-	tcpListener, err := net.Listen("tcp", webscocketServerAddress+wsc.port)
+	tcpListener, err := net.Listen("tcp", websocketServerAddress+wsc.port)
 	if err != nil {
 		return nil, err
 	}
 
 	var serveMux http.ServeMux
 
-	serveMux.HandleFunc(path.Join("/", rpcPath), wsc.request)
-	serveMux.HandleFunc(path.Join("/", rpcPath, "subscribe"), wsc.subscribe)
+	serveMux.HandleFunc(apiVersionPath, wsc.request)
```

Comment:
Instead of listening on the specific api version path (i.e. `/api/v1`), should we listen to the entire path `/api`, then we can lean on the logic in `wsc.request` to determine if this is a supported api version or not by parsing the url?